build/charts/antrea/crds/antreaagentinfo.yaml

```diff
@@ -13,7 +13,99 @@ spec:
       schema:
         openAPIV3Schema:
           type: object
-          x-kubernetes-preserve-unknown-fields: true
+          properties:
+            version:
+              type: string
+            agentConditions:
+              type: array
+              items:
+                type: object
+                required:
+                - type
+                - status
+                - lastHeartbeatTime
+                properties:
+                  lastHeartbeatTime:
+                    type: string
+                    format: date-time
+                  type:
+                    type: string
+                    enum: ['AgentHealthy', 'ControllerConnectionUp', 'OVSDBConnectionUp', 'OpenflowConnectionUp']
+                  status:
+                    type: string
+                    enum: ['True', 'False', 'Unknown']
+                  reason:
+                    type: string
+                  message:
+                    type: string
+            localPodNum:
+              type: integer
+              format: int32
+            nodePortLocalPortRange:
+              type: string
+            nodeSubnets:
+              type: array
+              items:
+                type: string
+            apiCABundle:
+              type: string
+            ovsInfo:
+              type: object
+              properties:
+                version:
+                  type: string
+                bridgeName:
+                  type: string
+                flowTable:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
would this work here instead:
```yaml
flowTable:
  type: object
  additionalProperties:
    type: number
    format: int32
```

Reply:
updated. Number represents `floating point numbers`. I think it should be `type: integer`?